jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -12,31 +61,42 @@ def step000_provision_sut(SKIP_PROVISIONING, script_dir) {
     }
 }
 
-def step010_setup_beaker(script_dir) {
-    withEnv(["SUT_HOST=${SUT_HOST}"]) {
+def step010_setup_beaker(script_dir, server_version) {
+    if (server_version["type"] != "pe") {
+        error "Unsupported server type: ${server_version["type"]}"
+    }
+
+    withEnv(["SUT_HOST=${SUT_HOST}",
+             "pe_version=${server_version["pe_version"]}",
```

Comment:
Ok, will that mean that for now that we can't install a non-released (i.e., mergely) version of PE?  Sorry, I don't really understand how these variables are used.